command/api/token/create.go

```diff
@@ -21,7 +21,7 @@ func createCommand() cli.Command {
 		Name:   "create",
 		Action: cli.ActionFunc(createAction),
 		Usage:  "create a new token",
-		UsageText: `**step api token create** <team-id> <crt-file> <key-file>
+		UsageText: `**step api token create** <team-id>|<team-slug> <crt-file> <key-file>
```

Comment:
I would use just <team> and use the text in the positional arguments to actually describe what that argument means.
```suggestion
		UsageText: `**step api token create** <team> <crt-file> <key-file>
```

Reply:
a1f4fb76